test/support/utils_sycl_defs.h

```diff
@@ -0,0 +1,64 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// This file contains SYCL-specific macros and abstractions to support different versions of SYCL library
+
+#ifndef _UTILS_SYCL_DEFS_H
+#define _UTILS_SYCL_DEFS_H
+
+#if __has_include(<sycl/sycl.hpp>)
```

Comment:
As far I notice that file duplicated the stuff from "../../../pstl/hetero/dpcpp/sycl_defs.h", which is an "entry point" for pulling  SYCL API and defining some oneDPL wrappers on top of SYCL API.
I would not have a huge code duplication here... I would propose to think  whether can we share that code between OneDPL impl and oneDPL tests?

Reply:
Normally, tests should not use any implementation details. Code duplication for simple utilities is a lesser problem than making most/all the tests depending on a particular implementation.